pages/downloads.md

```diff
@@ -116,23 +116,6 @@ For convenience, [MD5][MD5SUM_17] and [SHA1][SHA1SUM_17] hashes are also availab
 * [Examples][EXAMPLES_17]
 * [Javadoc][JAVADOC_17]
 
-### 1.6.6
```

Comment:
If 1.6.6 is being removed from the downloads page, it should also be removed from the Releases dropdown in the navbar.  This can be done by editing `_includes/nav.html`